### PR TITLE
Add a primary mixin, and replace other mixins with placeholder mixins

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

**Comment by notlee** [comment]
This looks good! Maybe we should rename `--standard` to `--editorial`, since that style seems to be used in article pages?

**Comment by notlee** [comment]
Oh! Other primary mixins haven't included base in the options map. The base is included by default as it's needed for all options and we're encouraging a single call to the primary mixin within a project. Is there a reason for adding it here?

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to bc0679c44c81fddd785b19834aa2c6040c9e4e06
No significant bundle size differences found.

File: main.scss [hunk 1]
```diff
@@ -3,7 +3,66 @@
 @import "o-typography/main";
 @import "o-icons/main";
 
-@import 'src/scss/variables';
+@import "src/scss/variables";
 @import "src/scss/color-use-cases";
+
+// ensure that the placeholders are only output once
+$_o-quote-placeholders-defined: false !default;
 @import "src/scss/base";
```
Comment (notlee) [inline main.scss]: Hmm I worry wrapping imports in variables might be prone to refactor whoopsies. It's not clear that those files should contain placeholders only. If I remember right Bren tried to bring placeholders back before, until n-ui css splitting ruined the fun. Their approach was to have a global list of placeholders which had been output and perform the check within the mixin which does the extending. I think it was a good approach as it keeps the placeholder and the control for outputting that placeholder once in the same place. 

I think it was something like:
```scss
@mixin oQuoteBaseCite {
    // Output the placeholder if it hasn't been output already.
    @if (not index($_o-quote-placeholders-defined, 'o-quote-cite')) {
        %_o-quote-cite {

        }
        $_o-quote-placeholders-defined: append($_o-quote-placeholders-defined, 'o-quote-cite');
    }

    // Extend the o-quote placeholder.
    @extend %_o-quote-cite;
}
```

What do you think?

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to 763ec6ff0442914b4671263235dea7e83b4afd36
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to 5fbd2460f4f3d256353179a7b000f4c0e0784bd2
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to ec27d15233829bb1139f293a1303404d8ea83720
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to 53fc31902fb3a94da123c7a59c885587d94118f7
No significant bundle size differences found.

**Comment by notlee** [comment]
Shhh fox

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to 893f9a5ed97763df84287d500697a8c4450bd970
No significant bundle size differences found.

**Comment by notlee** [comment]
Shhh friend

File: main.scss (hunk 1 above)
Comment (chee) [reply main.scss]: yeah, that seems good. i'll do it that way.

**Comment by origamiserviceuser** [comment]
o-quote bundle size difference from 2.2.24 to b2197c4584bc439d376c594c7cbff77c3ecfb677
No significant bundle size differences found.

**Pull request merged**